assets/app/lib/hex.rb

```diff
@@ -30,6 +30,7 @@ module Hex
       black: '#000000',
       red: '#ec232a',
       blue: '#35A7FF',
+      purple: '#800080',
```

Comment:
This allows users to config the color, because user.rb iterates over Lib::Hex::COLOR. The config wouldn’t have any effect though (not saved, not accessed in view/game/hex). It works, but isn’t ideal.
So I’d either remove this line and change "purple" to hex value in 18CZ’s and 1873’s config or render frame in user selected color. The latter would entail using named colors for possible tile frames in the future, too.

Reply:
Moved to tile config in game classes